dspace-services/src/test/java/org/dspace/servicemanager/spring/TestSpringServiceManager.java

```diff
@@ -1,229 +0,0 @@
-/**
- * The contents of this file are subject to the license and copyright
- * detailed in the LICENSE and NOTICE files at the root of the source
- * tree and available online at
- *
- * http://www.dspace.org/license/
- */
-package org.dspace.servicemanager.spring;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
-
-import java.util.List;
-
-import org.dspace.servicemanager.MockServiceManagerSystem;
-import org.dspace.servicemanager.SampleAnnotationBean;
-import org.dspace.servicemanager.config.DSpaceConfigurationService;
-import org.dspace.servicemanager.example.ConcreteExample;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-
-
-/**
- * Testing the spring based service manager
- *
- * @author Aaron Zeckoski (azeckoski @ gmail.com)
- */
-public class TestSpringServiceManager {
```

Comment:
@mwoodiupui : Instead of deleting all these tests, should we move some/most over to `DSpaceServiceManagerTest`? These tests seem useful to me, as they prove that we are successfully able to register/deregister services with Spring, etc.

It seems like the overarching goal of this PR is to merge `SpringServiceManager` and `DSpaceServiceManager` into a single class (which I like, as it is a nice simplification).  So, that implies to me that we should also try to merge the test classes together.